api/yelp_beans/send_email.py

```diff
@@ -79,7 +76,7 @@ def send_batch_weekly_opt_in_email(meeting_spec):
     load_secrets()
     create_url = 'https://{}.appspot.com/meeting_request/{}'.format(
         secrets["PROJECT"],
-        meeting_spec.key.urlsafe()
+        meeting_spec.id
```

Comment:
good catch. I rewrote this file on my side to use SES instead of sendgrind and forgot about these changes